modules/apps/27-interchain-accounts/keeper/keeper.go

```diff
@@ -58,17 +58,7 @@ func NewKeeper(
 }
 
 // SerializeCosmosTx marshals data to bytes using the provided codec
-func (k Keeper) SerializeCosmosTx(cdc codec.BinaryCodec, data interface{}) ([]byte, error) {
-	msgs := make([]sdk.Msg, 0)
-	switch data := data.(type) {
-	case sdk.Msg:
-		msgs = append(msgs, data)
-	case []sdk.Msg:
-		msgs = append(msgs, data...)
-	default:
-		return nil, types.ErrInvalidOutgoingData
-	}
-
+func (k Keeper) SerializeCosmosTx(cdc codec.BinaryCodec, msgs []sdk.Msg) ([]byte, error) {
```

Comment:
We should probably move `SerializeCosmosTx` to live alongside `DeserializeCosmosTx` in relay.go or maybe move them both to a codec.go file or something! 
I guess that can be dealt with in a follow up PR

Reply:
I think we should just move it to the types package #458. I prefer to leave moving code around for followup prs since the diffs can be harder to verify if you move code and modify it. But if you just move code, then it is an easier burden of reviewing

Reply:
Ah nice! Missed that issue. Sounds good!